src/tools/web/client/src/components/Configuration.jsx

```diff
@@ -118,7 +127,7 @@ const Configuration = ({
     getKey: getCorrectKey,
     setKey: setCorrectKey,
     deleteKey: deleteCorrectKey,
-  }, tree)
+  }, { 'user': (tree && tree.user) || {} }) // only show user/ namespace
```

Comment:
I decided that elektra-web should focus on the editing of user configuration values.

I had an issue where validation wouldn't run, because I was running everything on the namespace (and not with `/` prefix). That's why I decided to only show the `user/` namespace and then prefix all set operations with `/` to make the validation run.

I also don't think that editing metadata/specifications should be the focus of elektra-web. The idea is to have an existing configuration/specification and edit values there.

While editing of metadata could be easily added later (the API is still there, I'm just only using `user/` and `/`), the focus of this project should be the interactive configuration tree.